src/helpers/credential.rs

```diff
@@ -1100,6 +1041,8 @@ mod tests {
                 None,
                 None,
                 None,
+                subject_id,
+                "[1]",
             )
             .await
         {
```

Comment:
Can we adjust the tests to check for the required revealed statements, so that they are correctly used within the request / offer / credential?

Reply:
added some asserts for checking required_revealed_index